helpers.c

```diff
@@ -313,4 +313,166 @@ hsl_to_rgb_float (float *h_, float *s_, float *l_)
   *l_ = b;
 }
 
+// RGB to RYB
+// adapted from http://www.deathbysoftware.com/colors/index.html
+	/**********************************************************************************************
+	 * Given a RYB color, calculate the RGB color.  This code was taken from:
+	 * 
+	 * @param iRed     The current red value.
+	 * @param iYellow  The current yellow value.
+	 * @param iBlue    The current blue value.
+	 * 
+	 * http://www.insanit.net/tag/rgb-to-ryb/
+	 * 
+	 * Author: Arah J. Leonard
+	 * Copyright 01AUG09
+	 * Distributed under the LGPL - http://www.gnu.org/copyleft/lesser.html
+	 * ALSO distributed under the The MIT License from the Open Source Initiative (OSI) - 
+	 * http://www.opensource.org/licenses/mit-license.php
+	 * You may use EITHER of these licenses to work with / distribute this source code.
```

Comment:
[This post](http://softwareengineering.stackexchange.com/questions/316393/what-to-do-when-using-mit-licensed-code-in-a-gpl-licensed-project) seems to clarify this scenario from the perspective of Joshua Gentry (AGPL).  I'm pretty much positive it should be fine to treat this like MIT/ISC.